weblate_web/static/style.css

```diff
@@ -4024,6 +4024,472 @@ input.fullwidth {
+  padding-bottom: 0px !important;
+}
+
+.pt-1 {
+  padding-top: 22px !important;
+}
+
+.style6 > div {
+  font-size: 18px !important;
+  line-height: 30px !important;
+  color: #2c3548;
+  padding-top: 15x;
+  padding-bottom: 20px;
+}
+
+.style2 t {
+  font-size: 16px;
+  line-height: 30px;
+  padding-left: 21px;
+  font-weight: 600;
+  letter-spacing: 0;
+}
+
+.pl-1 {
+  font-size: 16px;
+  padding-left: 20px;
+}
+
+/* Services CSS (Days check color) */
+
+.grey {
+  color: #bfc3c7 !important;
+  font-size: 16px;
+}
+
+.orange {
+  color: #f6664c !important;
+  font-size: 16px;
+}
+
+/* For change in font size of payments */
+.payment-font-size {
+  font-size: 16px !important;
+}
+
+.payment-font-size a {
+  padding-left: 40px;
+  text-decoration: none !important;
+}
+
+.payment-font-size a img {
+  position: relative;
+  top: 1px;
+  padding-left: 25px;
+}
+
+/* User.html CSS */
+
+/* User.html CSS(Service Title) */
+
+.updated-service-title span {
+  color: #00a481;
+  font-size: 16px;
+  margin-left: 40.5px;
+}
+
+.updated-service-title span a {
+  color: #00a481;
+  font-size: 16px;
+  letter-spacing: 0;
+  line-height: 20px;
+}
+
+.updated-service-title span img {
+  height: 15px;
+  width: 10px;
+  position: relative;
+  top: 3px;
+  margin-left: 5px;
+}
+
+/* User.html CSS(Donation Box) */
+
+.payment-form-donationbox {
+  height: 100px;
+  width: 100%;
+  max-width: 530px;
+  border-radius: 20px;
+  background-color: #ffffff;
+  box-shadow: 0 2px 10px 0 rgba(0, 0, 0, 0.1);
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.payment-form-donationbox a {
+  min-width: 0px;
+}
+
+.line-right-donationbox {
+  font-size: 18px;
+  padding-right: 30px;
+}
+
+/* User.html CSS(Line-left and Line-Right Custom Variants) */
+
+.line-left2 {
+  display: flex;
+  font-size: 40px !important;
+  letter-spacing: -1px;
+  line-height: 44px;
+  width: unset !important;
+  justify-content: space-between;
+  font-weight: unset !important;
+  color: #2c3647;
+}
+
+.left-line3 {
+  display: flex;
+  justify-content: space-between;
+  border: none;
+  margin-bottom: 20px;
+}
+
+.line-right2 {
+  color: #3f85ff;
+  font-size: 18px;
+  font-weight: 600;
+  letter-spacing: 0;
+  line-height: 23px;
+  margin-top: 17px;
+  margin-right: 40px;
+}
+
+.inner-content {
+  display: flex;
+}
+
+.inner-content-child1 {
+  width: 64.84%;
+}
+
+.inner-content-child2 {
+  width: 35.16%;
+}
+
+/* User.html CSS(Custom Styles and Custom Margin and Paddings) */
+
+.stylep {
+  padding-bottom: 0px !important;
+}
+
+.style2 > div {
+  font-size: 18px !important;
+  line-height: 30px !important;
+  color: #2c3548;
+  padding-top: 15x;
+  padding-bottom: 20px;
+}
+
+.style4 {
+  width: 167px !important;
+}
+
+.style3 {
+  width: unset !important;
+}
+
+.style3 span {
+  font-weight: bold;
+}
+
+.custom-gradient {
+  height: 20px;
+  background: linear-gradient(180deg, #e9eaec -45%, #ffffff 100%);
+  border: none;
+}
```

Comment:
I'd like to see less changes in the CSS as I hoped it would be possible to reuse some of the existing classes. Did you look into that?

Reply:
I did look into that